simtools/applications/production_calculate_resource_estimates.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python3
+
+r"""
+Estimate the required computing resources for a simulation based on command-line arguments.
```

Comment:
Remove 'based on command-line arguments'. Not sure what you mean with that? All simtools are configured through the command line.